desktop/apps/auction/routes.coffee

```diff
@@ -48,7 +48,6 @@ setupUser = (user, auction) ->
 
   artworks.comparator = (artwork) ->
     saleArtwork = artwork.related().saleArtwork
-    saleArtwork.get('lot_number') or
```

Comment:
Technically you should be sorting by `position` and then by `lot_label`. The reason is that `position` is still set to `lot_number.to_f` and doesn't do anything fancy to determine a float that better represents lot number suffixes like `a`, `b`, etc.

Does that necessitate a change here?